src/components/footer/Footer.tsx

```diff
@@ -66,26 +66,48 @@ export const Footer = () => {
 
                 <div className="menu">
                     <span className="title">{MENU}</span>
-                    <Link to={teamPageRoute}>{REPORTING}</Link>
-                    <Link to={page2Route}>{STORE}</Link>
-                    <Link to={page2Route}>{HOW_TO_SUPPORT}</Link>
-                    <Link to={page2Route}>{STORIES_OF_VICTORIES}</Link>
+                    <Link to={teamPageRoute} className="disable">
+                        {REPORTING}
+                    </Link>
+                    <Link to={page2Route} className="disable">
+                        {STORE}
+                    </Link>
+                    <Link to={page2Route} className="disable">
+                        {HOW_TO_SUPPORT}
+                    </Link>
+                    <Link to={page2Route} className="disable">
+                        {STORIES_OF_VICTORIES}
+                    </Link>
                 </div>
 
                 <div className="about_us">
                     <span className="title">{ABOUT_US}</span>
                     <Link to={teamPageRoute}>{ABOUT_US}</Link>
-                    <Link to={page2Route}>{OUR_HISTORY}</Link>
-                    <Link to={page2Route}>{OUR_TEAM}</Link>
-                    <Link to={page2Route}>{PARTNERS}</Link>
-                    <Link to={page2Route}>{EVENTS_AND_NEWS}</Link>
+                    <Link to={page2Route} className="disable">
+                        {OUR_HISTORY}
```

Comment:
Suggestion (non blocking):
It woulkd be good to have condition using classNames that will determine if elements is disabled instread of setting it like this ("hardcore value")

Reply:
Maybe in a future updates